src/verbs/queue_pair.rs

```diff
@@ -58,6 +57,24 @@ impl From<u32> for QueuePairState {
     }
 }
 
+#[bitmask(u64)]
+#[bitmask_config(vec_debug)]
+pub enum SendOperationType {
```

Comment:
Maybe `SendOperationFlags` is a better name since the original enum is called `ibv_qp_create_send_ops_flags`